### PR TITLE
PMM-7178 Rename columns. Remove ppg from matrix.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7178

### CODE REVIEW

**Review by JiriCtvrtka — APPROVED** [review]

File: agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 1]
```diff
@@ -239,7 +239,6 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 					IsTruncated:         currentPSM.IsQueryTruncated,
 					Fingerprint:         currentPSM.Fingerprint,
 					Database:            currentPSM.Database,
-					Tables:              currentPSM.TablesNames,
```
Comment (BupycHuk) [inline agents/postgres/pgstatmonitor/pgstatmonitor.go:242]: TablesNames was renamed to Relations, so please rename it instead of removing this field.

Comment (BupycHuk) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:242]: https://github.com/percona/pg_stat_monitor/blob/master/docs/USER_GUIDE.md

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: agents/postgres/pgstatmonitor/pgstatmonitor.go (hunk 1 above)
Comment (askomorokhov) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:242]: Oh, thank you! I looked for it but not found.

Comment (askomorokhov) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:242]: Renamed.

Comment (BupycHuk) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:242]: Should we rename `MetricsBucket_Common` field?

File: agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 2]
```diff
@@ -239,7 +239,7 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
...
+					Relations:           currentPSM.Relations,
```
Comment (BupycHuk) [inline agents/postgres/pgstatmonitor/pgstatmonitor.go:242]:
```suggestion
					Tables:           currentPSM.Relations,
```

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**